TopMovies/Service/MovieService.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MovieService.swift
+//  TopMovies
+//
+//  Created by anikolaenko on 02.02.2021.
+//
+
+import Foundation
+
+struct MovieService {
```

Comment:
Add `StoreSurbscriber` conformance.

And when a state is going to be `requested`, service should send action `.downloading` and perform an API call.